server/models/dataset_helpers/image_metadata.py

```diff
@@ -361,7 +361,8 @@ def transform(cls, value):
                 cls._assertEnumerated(value, {
                     'histopathology',
                     'serial imaging showing no change',
-                    'single image expert consensus'})
+                    'single image expert consensus',
+                    'confocal with consensus dermoscopy'})
```

Comment:
Should the new value be added to the single image upload tool as well?

Reply:
Good point.

Reply:
Fixed.